client/modules/product-variant/templates/products/productDetail/productDetail.js

```diff
@@ -85,6 +94,7 @@ Template.productDetail.helpers({
                     handle: result
                   });
                 }
+                return null;
```

Comment:
Just a question. Why do we return `null` here instead of `undefined` like we do elsewhere?